src/plugins/console/server/lib/request.ts

```diff
@@ -37,7 +37,7 @@ interface Args {
 // We use a modified version of Hapi's Wreck because Hapi, Axios, and Superagent don't support GET requests
 // with bodies, but ES APIs do. Similarly with DELETE requests with bodies. Another library, `request`
 // diverged too much from current behaviour.
-export const sendRequest = ({
+export const proxyRequest = ({
```

Comment:
@joshdover I'm wondering if the platform should provide proxying functionality out of the box. The current implementation stops working when we merge https://github.com/elastic/kibana/issues/55670 Should I re-open https://github.com/elastic/kibana/issues/41960 ?

Reply:
Makes sense to me, but I think we should let this PR move forward and refactor it once we have a solution for #41960.

IMO, we could even push this work off until after NP migrations are complete, but before 8.0.

Reply:
works for me